MD/md-26/README.md

```diff
@@ -0,0 +1,87 @@
+# MD-26: User-facing checks
+- **Description**: Automate checking user experience by submitting transactions every few minutes
+- **Authors**: [Andy Golay](mailto:liam@movementlabs.xyz)
+
+
+<!--
+  This template is for drafting Desiderata. It ensures a structured representation of wishes, requirements, or needs related to the overarching objective mentioned in the title. After filling in the requisite fields, please delete these comments.
+
+  Note that an MD number will be assigned by an editor. When opening a pull request to submit your MD, please use an abbreviated title in the filename, `md-draft_title_abbrev.md`.
+
+  TODO: Remove this comment before finalizing.
+-->
+
+## Overview
+We need to check that users can submit transactions. This MD outlines a simple automated solution.
+
+## Definitions
+
+Provide definitions that you think will empower the reader to quickly dive into the topic.
+
+## Desiderata
+
+  ### Desideratum Title
```

Comment:
Provide title and number: https://github.com/movementlabsxyz/MIP/blob/ee41cb22e5039d5e04cb65638a53202177d5ffe7/MD/md-0/README.md?plain=1#L46